js/foam/apps/calc/Calc.js

```diff
@@ -175,7 +190,7 @@ CLASS({
       name: "mult",
       label: "×",
       speechLabel: "multiply",
-      keyboardShortcuts: [ "*" ],
+      keyboardShortcuts: [ "*", "x" ],
```

Comment:
Allows users to use 'x' to specify the multiply command, as per request of a11y folks